src/pages/resources/reference/push-notifications/templates/index.md

```diff
@@ -14,17 +14,17 @@ This document outlines the push templates that are supported by default.
 
 <InlineAlert variant="info" slots="text"/>
 
-Default push template functionality is available for use with the **Adobe Campaign Classic** extension.
+Default push template functionality is available for use with the **Adobe Campaign Classic** extension. In additon, the rich push template functionality is supported starting Campaign v8. 
```

Comment:
I think "... is supported starting with Campaign v8." is easier to read. In addition, should we say Campaign Classic v8 instead of Campaign v8? IMO, our docs here cover the Campaign Standard extension as well and this may lead to some confusion. This applies to the naming changes below as well.

Reply:
I agree with saying "... starting _with_ Campaign v8."

Reply:
thank you @rymorale and @calebk1 for your feedback. I updated as you suggested. Campaign Classic is only for v7. Starting v8, we should only refer to Adobe Campaign.